openedx/core/djangoapps/coursegraph/management/commands/dump_to_neo4j.py

```diff
@@ -84,6 +85,8 @@ def serialize_item(item):
         fields['location'] = six.text_type(item.location)
 
         block_type = item.scope_ids.block_type
```

Comment:
Looks good, just need some reordering of lines. I will suggest to move `block_type` initialization next to `course_key` initialization and also `fields['block_type']` and `fields['detached']` next to block where `fields` is being populated so these two new will be covered under same comment given to this block.